stream_alert_cli/terraform/generate.py

```diff
@@ -340,20 +340,17 @@ def terraform_generate(**kwargs):
     Returns:
         bool: Result of cluster generating
     """
-    config = kwargs.get('config')
-    init = kwargs.get('init', False)
-
     cleanup_old_tf_files()
 
     # Setup the main.tf.json file
     LOGGER_CLI.debug('Generating cluster file: main.tf.json')
-    main_json = json.dumps(
-        generate_main(init=init, config=config),
-        indent=2,
-        sort_keys=True
-    )
     with open('terraform/main.tf.json', 'w') as tf_file:
-        tf_file.write(main_json)
+        json.dump(
```

Comment:
can we decide on an indentation style? we seem to be all over the place:

```
json.dump(arg,
          arg2,
          arg3)

or

json.dump(
  arg,
  arg2,
  arg3
)
```

Reply:
I'm not seeing anywhere that uses the first format? there is a one-liner with `json.dump(...)` but can't find anywhere that uses the format:
```
json.dump(arg,
          arg2,
          arg3)

```

Reply:
I'm not speaking to just `json.dump`, just speaking generally

Reply:
Does python[ PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation) answer this question?
```
Yes:

# Aligned with opening delimiter.
foo = long_function_name(var_one, var_two,
                         var_three, var_four)

# More indentation included to distinguish this from the rest.
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    print(var_one)

# Hanging indents should add a level.
foo = long_function_name(
    var_one, var_two,
    var_three, var_four)
```

Reply:
So is that saying they are all acceptable?

Reply:
Yes. I prefer styles which don't require changes if the function were renamed (i.e. I don't like the first one in the PEP8 list)